<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccelCalibrationOfIndependent</class>
 <widget class="QWidget" name="AccelCalibrationOfIndependent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>加速度计校准</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>490</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>校准</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_declare">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>561</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>让机身水平并设置默认加速偏移值 (3 axis)这个需要您将机器放到每个边缘的位置.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_tipPicture">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>181</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_tipText">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_cutdownText">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>150</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
